<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Procese</class>
 <widget class="QDialog" name="Procese">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addproces">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adauga un nou proces</string>
   </property>
  </widget>
  <widget class="QPushButton" name="iesire">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>500</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Iesire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Predefinit sunt afisate toate procesele pentru toate agentiile.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Daca se doreste afisarea doar pentru una singura, selectati din lista de mai jos agentia dorita</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="newp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pentru adaugarea unui noi proces, completati campurile urmatoare:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tip">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>138</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tipul procesului:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tipBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Transport rutier</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Transport aerian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Transport maritim</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Transport feroviar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Transport special</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Depozitare</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="nume">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nume responsabil:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prenume">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prenume responsabil:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numeEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenumeEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="agentie">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agentia corespunzatoare:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="agentieBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>240</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anuleaza">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>290</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anuleaza</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
